app/api/utils.py

```diff
@@ -438,19 +438,23 @@ def parse(self, file):
         file = EncodedIO(file)
         file = io.TextIOWrapper(file, encoding=file.encoding)
         data = []
-        for _, line in enumerate(file, start=0):
+        for i, line in enumerate(file, start=0):
             if len(data) >= settings.IMPORT_BATCH_SIZE:
                 yield data
                 data = []
             j = {'text': '', 'labels' :[]} 
```

Comment:
no initialization necessary here. create the dict directly at the `data.append()` line.